src/components/calendarfunc.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import Calendar from "react-calendar";
+import "./Calendar.css";
+import Checklist from "./checklist";
+import MemoFunc from "./memo";
+
+import Dropdown from "react-bootstrap/Dropdown";
+
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const CalendarFunc = () => {
+  const [selectedDate, setSelectedDate] = useState(null);
+  const [eventName, setEventName] = useState("");
+  const [events, setEvents] = useState([]);
+
+  //States for Checklist/Memo display.
+  const [showChecklist, setShowChecklist] = useState(false);
+  const [showMemo, setShowMemo] = useState(false);
+
+  // Function to select the date.
+  const handleDateSelection = (date) => {
+    setSelectedDate(date);
+  };
+
+  // Function to handle Input Change.
+  const handleEventInputChange = (event) => {
+    setEventName(event.target.value);
+  };
+
+  // Allows users to create a new event.
```

Comment:
This comment here is okay. It tells me more about the app functionality rather than just what the function is doing. If it was only `Creates new event`, that would be redundant then.